client/src/components/SudoModePasswordField/SudoModePasswordField.js

```diff
@@ -52,12 +52,16 @@ function SudoModePasswordField(props) {
     const headers = {
       'X-SecurityID': Config.get('SecurityID'),
     };
-    const responseJson = await fetcher(data, headers);
-    if (responseJson.result) {
-      onSuccess();
-    } else {
-      setResponseMessage(responseJson.message);
-    }
+    fetcher(data, headers)
+      .then(responseJson => {
+        if (responseJson.result) {
+          onSuccess();
+        }
```

Comment:
What is the condition where this is false? Is that some sort of uncaught error?

Reply:
Actually the `result` bool is actually kind of redundant as in CMS 6 it's fully use status codes - https://github.com/silverstripe/silverstripe-admin/blob/3.0/code/SudoModeController.php#L97 - this wasn't the case in CMS 5

I'll simply remove the check for 'result' and remove it from the endpoint